crates/webidl/Cargo.toml

```diff
@@ -12,14 +12,21 @@ name = "webidl-expected"
 path = "tests/expected/lib.rs"
 
 [dev-dependencies]
+diff = "0.1.11"
+env_logger = "0.5.10"
 wasm-bindgen = { version = "=0.2.11", path = "../..", default-features = false }
 wasm-bindgen-backend = { version = "=0.2.11", path = "../backend", features = ["extra-traits"] }
-diff = "0.1.11"
+# Force the `log` dependency for testing.
+wasm-bindgen-webidl = { version = "=0.2.11", path = ".", features = ["log"] }
```

Comment:
Oh dear now that's something that was never intended to be allowed in Cargo... Ah well!